solutions/IOI 13-game.mdx

```diff
@@ -28,6 +28,15 @@ because they support point updates and range queries. Each segment tree node
 stores a treap, and updating a node involves changing $\mathcal O(\log C)$ values
 in its treap (similar to updating a 2D segment tree node).
 
+<Optional>
+
+It's possible (and probably intended) to solve this problem using a segment trees
```

Comment:
It \*is\* intended. I'm pretty sure I remember reading about this, though I can't find it ...

Preferably include such a solution